src/components/ccsds_packetizer/gen/templates/assembly/name_ccsds_cosmos_telemetry.txt

```diff
@@ -0,0 +1,25 @@
+{% set plugin_format_dictionary = {'F': 'FLOAT', 'U': 'UINT', 'E': 'UINT', 'I': 'INT'} %}
+{% for id, packet in packets.items() %}
+Telemetry {{ name }} {{ packet.name }} Big_Endian "{{ packet.description }}"
+{# The ccsds packet header #}
+{% for field_name, field in ccsds_primary_header_model.fields.items() %}
+  Item {{ field.name }} {{ field.start_bit }} {{ field.size }} {{ plugin_format_dictionary[field.format.type[0]] }} "{{ field.description|replace('\n','') }}"
+{% if field.is_enum %}
+{% for literals in field.datatype.model.literals %}
+    State {{ literals.name }} {{ literals.value }}
+{% endfor %}{% endif -%}{% endfor %}
+  Item Ccsdssec 48 32 UINT "Seconds since epoch"
+  Item Ccsdssubsec 80 32 UINT "Milliseconds of day (0 - 86399999)"
```

Comment:
Instead of declaring these statically, it would be better to use the `sys_time.record.yaml` model instead, similarly to how you have done it above with the `ccsds_primary_header_model`.

Reply:
Addressed in [384ba33](https://github.com/lasp/adamant/pull/21/commits/384ba335a92093d290c490b015b3f3e44161f1bc).